src/app/modules/form/admin-form/admin-form.service.ts

```diff
@@ -554,3 +558,59 @@ export const updateFormSettings = (
     return okAsync(updatedForm.getSettings())
   })
 }
+
+/**
+ * Deletes form logic.
+ * @param form The original form to delete logic in
+ * @param logicId the logicId to delete
+ * @returns ok(true) on success
+ * @returns err(database errors) if db error is thrown during logic delete
+ * @returns err(LogicNotFoundError) if logicId does not exist on form
+ */
+export const deleteFormLogic = (
```

Comment:
can we add unit tests for this function too?

Reply:
added